pkg/usagemetrics/googleanalytics.go

```diff
@@ -0,0 +1,79 @@
+package usagemetrics
+
+import (
+	"github.com/golang/glog"
+	analytics "github.com/jpillora/go-ogle-analytics"
+	k8sapi "github.com/openebs/maya/pkg/client/k8s/v1alpha1"
+	openebsversion "github.com/openebs/maya/pkg/version"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	k8sversion "k8s.io/client-go/pkg/version"
+)
+
+const (
+	// GAclientID is the unique code of the project in Google Analytics
+	GAclientID = "UA-127388617-1"
+)
+
+// Event is a custom metric collected by OpenEBS developers
+type Event struct {
+	category string
```

Comment:
We must provide comments that describe each property of Event.

Reply:
I am not able to understand any of the properties without a proper description for each.